FALSEHOODS.md

```diff
@@ -118,3 +118,8 @@ surprising how many false assumptions programmers continue to make about them.
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **Phone numbers have only one prefix**
+
+    In the mid-90s in Iceland, phone numbers changed from 5 and 6 digits to 7 digits.
+    During the transition period, phone numbers could be reached by the old area code
+    or the new 7 digit number (a different prefix).
```

Comment:
According to an old news article (in Icelandic) the transition begun in one region before the others (I can't verify this anymore) so applying automated updates to entire databases is not a good idea; at least not during the transition period.  It also means if you're looking for a record by phone number, you have to consider both prefixes.

If you are writing an auto dialing tool for, say helpdesk operators, you have to be aware of which region is available through the new seven digit number, or the old regional code, or both.